test/e2e/scenarios/Cluster/cluster_create_update.yaml

```diff
@@ -29,6 +29,28 @@ steps:
           ACK.ResourceSynced:
             status: "True"
             timeout: 7200
+    expect_k8s:
+      spec:
+        name: cluster$RANDOM_SUFFIX
+        nodeType: db.t4g.medium
+        aclName: open-access
+        numShards: 1
+        description: Test create from ACK
+        snsTopicARN: $TOPIC1
+        numReplicasPerShard: 1
+        tlsEnabled: true
+        snapshotWindow: 05:00-06:00
+        snapshotRetentionLimit: 0
+    expect_aws:
+      Name: cluster$RANDOM_SUFFIX
+      NodeType: db.t4g.medium
+      ACLName: open-access
+      NumberOfShards: 1
+      Description: Test create from ACK
+      SnsTopicArn: $TOPIC1
```

Comment:
We need to assert status entries?

Reply:
Do you mean we need to add status assertion? If so, what fields do you want me to add? Or do you mean why we assert `NumberOfShards` (from status) here? Because describe_clusters API called by boto3 doesn't have `NumShards`. It only has `NumberOfShards`, so that is the only field to validate number of shards.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/memorydb/client/describe_clusters.html